### ISSUE TITLE
Question: how to set approperiately TinySA Ultra as real time waterfall for ICOM7100, like SDR receiver ?

### ISSUE THREAD

**Issue by Nokia808** [erikkaashoek/NanoVNA#69]
Hello dear.

1st of all, I would like to give developer my hot thanks for this great device which has really great testing abilities in compare to it's small size ! Really it is a complement for victor network analyzer SV4401A making very strong testing & lab tool !

However, we have difficulty to set it as fast real time with good visualization for useful signals waterfall for ICOM 7100 transceiver.

We modify ICOM 7100 by adding an output port for it's IF from official points for that in it's board. It work 100% with SDR receiver. However, I preferred to use TinySA Ultra for this purpose because I have no SDR receiver. SDR receiver is owned by my friend .....

We have difficulty to set TinySA Ultra as waterfall for ICOM 7100. The chief difficulties are:

1) it is slow. When we increase RBW it will become fast but the visualization of useful signal (it's carve will be smaller)
2) we need to set a marker (single marker) to point constantly to frequency that radio at at. We failed for that.

We tried center/span mode, then set span at 400 KHz. We make "Marker1" only active marker & set it's type as "Normal" then set "central" for this marker. But we did not get a marker point to frequency that the radio dealer set on to ...

I'm already opened a thread on www.qrz.com forum - see following link:
https://forums.qrz.com/index.php?threads/how-to-set-tinysa-to-make-real-time-spectroscpe-for-icom-7100.869913/
but we did not, yet, reach to solution .....

My call sign is: YI1YJK

Any help is appreciated.

Regards.

**Comment by Nokia808** [comment]
Issue fixed ! Please read the following:

Set TinySA Ultra as Real Time Spectroscope for Station:
Proceed as following step by step:
1) launch device on it's default setting,
2) go to "Marker" > select "Modify markers" > disable "tracking",
3) go to "Marker Options" > select "Center",
4) go to "Frequency" & perform the following:
- select "center" & enter center frequency of station (124.487 Khz for ICOM7100), then "Back" to "Frequency",
- "Frequency" > select "Span" & enter 200 KHz,
- leave "RBW" on "Auto" as it & never change it.
5) go to "Display" > select "Sweep accuracy" & perform the following:
- select "Fast",
- set "Speedup at 4,
- then "Back",
- select "Sweep points" & set it at 201
6) go to "Storage" > press on "Save setting"
7) connect "RF" port of device to "IF output" port of station,
8) enjoy real time spectroscope !

Important Notes:
A) you should never perform step (5) before steps (2), (3), & (4). Otherwise "Central Marker" will never be exactly equal to "Center Frequency" & will be slight shift from it. While performing step (5) after steps (1), (2), (3) & (4) in sequence will guarantee that "Center Marker" will exactly equal to center frequency of station & by this way will point constantly be at middle of screen of device pointing exactly to real frequency that you tune you station on it.
B) leave default state (only one marker, marker 1)
C) in "Trace" > "Trace 1", leave "Clac" at it's default state (OFF) or change it to "Quasi Peak". Otherwise speed of analyzer will be slow .... "OFF" state may be useful if your shack in a town or other environment where QRM is absent or low, while "Quasi Peak" is more useful where QRM is harsh. "Quasi Peak" is even able to show on screen the weak signal that you can not hear due to QSB attacks ....

------------------
Please look for attached video by me in the following link (my review on Alibaba.com - see "Buyer review" section in link):
https://www.alibaba.com/product-detail/ULTRA-100k-5-3GHz-3-95_1600697601084.html

I tested it on CW, SSB (LSB & USB) & AM modes & it worked !

YI1YJK
Regards.

**Comment by Nokia808** [comment]
I have to confess my wrong now ! Yesterday, I spent 3 hrs from monitoring of tiny spectrum analyzer Ultra to compare between:
- CALC OFF
- AVER 4
- QUASI PEAK

Really "Quasi Peak" is the best option ! It was, in fact, slopping of care that mislead me !
1) 1st of all, "Quasi Peak" not slower than "Calc OFF" at all.
2) 2nd thing, it remove noise background & make useful signal more prominent.
3) More importantly, I noticed that "Qausi Peak" is able to detect weak signal that CAN NOT HEAR !

I discovered 3rd point while QSB attacks happened during listening QSO on 40, 20, & 15 mb. When one of 2 operator has weaker signal but still could be heard without QSB, when QSB happened I was unable to listen to him but it's signal appearing on the spectrum screen ! Then, after more observation, I noticed while I monitored QSO that I listened to only one operator, I noticed signal appearing after he stopped as if 2nd operator replied him. Then when propagation improved temporarily, I could copied voice of 2nd operator that I suspected him from curve of analyzer !

"CALC OFF" is useful if your shack in a town or other quit environment with no or minimum QRM. But, if you - like me - live in harsh QRM environment like a vertical building then "Quasi Peak" should be your choice.

I edited my previous 2nd post to be compatible with this post.

I think it is now the time to close this issue. But, I would like from developer to pay more attention in future update to improve this feature (real time spectroscope monitoring) but remove slowing. At current stage, enabling waterfall make unwanted slowing, something of annoying bother .....
